src/views/catalog/wizard/tabs/scheduling/components/WizardSchedulingGrid.tsx

```diff
@@ -99,6 +100,21 @@ const WizardSchedulingGrid: React.FC<WizardSchedulingGridProps> = ({ updateVM, v
           />
 
           <WizardDescriptionItem
+            helperPopover={{
+              content: (
+                <Trans ns="plugin__kubevirt-plugin" t={t}>
+                  <p>
+                    The descheduler can be used to evict a running pod to allow the pod to be
+                    rescheduled onto a more suitable node.
+                  </p>
+                  <br />
+                  <p>
+                    Note: if VirtualMachine has LiveMigratable=False condition, edit is disabled.
+                  </p>
+                </Trans>
```

Comment:
this is already used in another place, consider making it its own component next to Descheduler component and consume it here and in VirtualMachineSchedulingLeftGrid

Reply:
Thank you :)